stable/database/files/backup_hooks.py

```diff
@@ -376,6 +473,15 @@ def start_server(port):
         LOGGER.info("Starting backup hooks server on port %s", port)
         httpd.serve_forever()
 
+def verify_prerequisites():
+    if FREEZE_MODE == "hotsnap":
+        if which("nuocmd") is None:
+            raise RuntimeError("'nuocmd' command not found")
+        elif subprocess.call(["nuocmd", "pause", "archiving", "-h"], 
+                           stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL) == 2:
```

Comment:
What if it returns 1? I think we should only check that the exit code equals 2 before saying that the command is not supported, otherwise we should propagate whatever error appeared on stdout/stderr.

Reply:
Will do.